views/partials/_cookie-banner.njk

```diff
@@ -0,0 +1,108 @@
+{% from "govuk/components/cookie-banner/macro.njk" import govukCookieBanner %}
+
+{% set html %}
+  <p>We’d like to use analytics cookies so we can understand how you use the Design System and make improvements.</p>
+  <p>We also use essential cookies to remember if you’ve accepted analytics cookies.</p>
+{% endset %}
+
+{% set acceptHtml %}
+  <p>You’ve accepted analytics cookies. You can <a class="govuk-link" href="/cookies">change your cookie settings</a> at any time.</p>
+{% endset %}
+
+{% set rejectHtml %}
+  <p>You’ve rejected analytics cookies. You can <a class="govuk-link" href="/cookies">change your cookie settings</a> at any time.</p>
+{% endset %}
+
+{{ govukCookieBanner({
+  ariaLabel: "Cookies on GOV.UK Design System",
+  hidden: true,
+  attributes: {
+    "data-module": "govuk-cookie-banner"
+  },
+  messages: [
+    {
+      headingText: "Cookies on GOV.UK Design System",
+      html: html,
+      actions: [
+        {
+          text: "Accept analytics cookies",
+          type: "button",
+          classes: "js-cookie-banner-accept"
+        },
+        {
+          text: "Reject analytics cookies",
+          type: "button",
+          classes: "js-cookie-banner-reject"
+        },
+        {
+          text: "View cookies",
+          href: "/cookies"
+        }
+      ],
+      classes: "app-width-container js-cookie-banner-message"
+    },
+    {
+      html: acceptHtml,
+      role: "alert",
+      hidden: true,
+      actions: [
+        {
+          text: "Hide this message",
+          classes: "js-cookie-banner-hide js-cookie-banner-hide--accept"
+        }
+      ],
+      classes: "js-cookie-banner-confirmation-accept"
+    },
+    {
+      html: rejectHtml,
+      role: "alert",
+      hidden: true,
+      actions: [
+        {
+          text: "Hide this message",
+          classes: "js-cookie-banner-hide js-cookie-banner-hide--reject"
+        }
+      ],
+      classes: "js-cookie-banner-confirmation-reject"
+    }
+  ]
+}) }}
+
+<!-- Inline script to show the cookie banner as soon as possible,
+to avoid a high cumulative layout shift (CLS) score https://web.dev/cls/ -->
+<script>
+  /* If cookie policy changes and/or the user preferences object format needs to
+  * change, bump this version up afterwards. The user should then be shown the
+  * banner again to consent to the new policy.
+  *
+  * Note that because isValidCookieConsent checks that the version in the user's
+  * cookie is equal to or greater than this number, you should be careful to
+  * check backwards compatibility when changing the object format.
+  *
+  */
```

Comment:
We might want to consider making these Nunjucks comment blocks – think they're most useful when making changes to the inline script, and probably not that useful in the outputted HTML / script? What do you reckon?

Reply:
@36degrees I've changed the comment just before the inline script to be a Nunjucks comment instead so it's not output. 

I'm not sure about the comments within the inline script itself, as I think they're still useful to explain the relationship between some of the variables/functions and cookie-functions.js, but making them Nunjucks comments will mean moving them out of the script and away from the relevant variable/function?

Reply:
You can still use Nunjucks comments mixed in with the JavaScript, and they'll get stripped out and won't appear in the outputted HTML. Although whether that's potentially confusing is another matter…

I don't have a strong preference either way, just thought it was worth considering :)